modules/tinymce/src/core/test/ts/browser/keyboard/HomeEndKeysTest.ts

```diff
@@ -53,6 +53,23 @@ describe('browser.tinymce.core.keyboard.HomeEndKeysTest', () => {
       TinyAssertions.assertSelection(editor, [ 0, 0 ], 1, [ 0, 0 ], 1);
     });
 
+    context('Inline element', () => {
```

Comment:
These aren't really inline elements, they are `Inline boundary elements` so maybe something like this?
```suggestion
    context('Inline boundaries', () => {
```